### ISSUE TITLE
Can't install @pixi/events plugin

### ISSUE THREAD

**Issue by sabigara** [inlet/react-pixi#333]
### Description

First of all, thank you for creating this awesome library! It's been helping me a lot.

I'm trying to use `@pixi/events` plugin ([doc](https://api.pixijs.io/@pixi/events.html)) with `react-pixi` but experiencing trouble.

We need to delete the default `interaction` plugin before installing `@pixi/events`:

```typescript
import { Renderer, Loader } from "pixi.js";

delete Renderer.__plugins.interaction;

const app = new PIXI.Application();
const { renderer } = app;

renderer.addSystem(EventSystem, 'events');
```

But `react-pixi` is trying to set the resolution of `interaction` at: https://github.com/inlet/react-pixi/blob/11e258e05f57a7609831125bb89b226bc685ccc6/src/stage/index.js#L211-L213

So an error is thrown.

It seems `react-pixi` only refers to `plugins.interaction` here so a simple existance check may be enough:

```typescript
resetInteractionManager() {
  if("interaction" in this.app.renderer.plugins) {
      this.app.renderer.plugins.interaction.resolution = this.app.renderer.resolution
  }
}
```

If this solution is OK, I would make a PR.

### Steps to reproduce

https://codesandbox.io/s/thirsty-cloud-cd17s3?file=/src/App.js

### Additional info

- `@inlet/react-pixi` version: 6.6.5
- `React` version: 17.0.2
- `ReactDOM` version: 17.0.2
- `PIXI` version: 6.3.0

**Comment by inlet** [comment]
Sounds like a good solution, yes please create a PR. Thanks!

**Comment by inlet** [comment]
Fixed in #335
